### PR TITLE
F6005: New way of changing the PLOAM password

### PR DESCRIPTION
**Pull request by diegoemy76**
i found a way to change ploam password editing the html code using developer option of the browser

### CODE REVIEW

**Comment by simonebortolin** [comment]
1) This pr does not respect the template and this information must go inside the ploam paragraph imported via the nokia or cig.md file

2) Even if this pr does respect the template I don't see the point, the web ui only mentions it if it is the only known way to change the ploam, as in the case of fritzbox.

**Comment by simonebortolin** [comment]
Ahh this work whitout change firmware

**Comment by diegoemy76** [comment]
Si si funziona con il firmware stock

Il dom 24 mar 2024, 09:34 Simone ***@***.***> ha scritto:

**Comment by simonebortolin** [comment]
Bon allora è da mettere nella sezione corretta e si può approvare la pr

**Comment by diegoemy76** [comment]
Okay aggiorno

Il dom 24 mar 2024, 09:37 Simone ***@***.***> ha scritto:

**Comment by nanomad** [comment]
> * This pr does not respect the template and this information must go inside the ploam paragraph imported via the nokia or cig.md file
> *

Does it makes sense though? It is something related to the ZTE custom UI and not CIG

File: _ont/ont-zte-f6005.md [hunk 1]
```diff
@@ -29,7 +29,9 @@ parent: ZTE
 {% include image.html file="f6005_of.jpg" alt="F6005 Open Fiber" caption="F6005 with Open Fiber branding <a href='https://forum.fibra.click/u/mirko991'>@mirko991</a>" %}
 {% include image.html file="f6005_tim.jpg" alt="F6005 TIM" caption="F6005 with ZTE branding, like the ones used by TIM" %}
 {% include image.html file="f6005_teardown.jpg" alt="F6005 teardown" caption="F6005 teardown" %}
-
+## Change Ploam password
```
Comment (nanomad) [inline _ont/ont-zte-f6005.md:32]:
```suggestion
## Change PLOAM password from the web UI
```

File: _ont/ont-zte-f6005.md [hunk 2]
```diff
@@ -29,7 +29,9 @@ parent: ZTE
...
+The Password and Update Password  fields are disabled and hidden, but can be re-enabled via the browser's developer console by editing the HTML code
```
Comment (nanomad) [inline _ont/ont-zte-f6005.md:33]:
```suggestion
On some software versions, Password and Update Password fields are disabled and hidden even when using the admin account. They sometimes can be re-enabled via the browser's developer console by editing the HTML code and setting:

- CSS display property to show (from none)
- HTML `disabled=true` attribute to `enabled=true`
```

File: _ont/ont-zte-f6005.md [hunk 3]
```diff
@@ -29,7 +29,9 @@ parent: ZTE
...
+Tested and fully working  with V6.0.10P2N19 (OpenFiber) firmware
```
Comment (nanomad) [inline _ont/ont-zte-f6005.md:34]:
```suggestion

This has been tested on the following firmware versions:
- V6.0.10P2N19 (OpenFiber)
```

**Review by nanomad — CHANGES_REQUESTED** [review]

**Review by nanomad — APPROVED** [review]

**Comment by nanomad** [comment]
@simonebortolin 
LGTM

image

**Comment by simonebortolin** [comment]
Ok, i dont like the image, but its ok

**Pull request merged**